src/tools/dev/scripts/bv_support/bv_qwt.sh

```diff
@@ -71,7 +71,7 @@ function bv_qwt_print
 
 function bv_qwt_print_usage
 {
-    printf "%-20s %s [%s]\n" "--qwt" "Build with Qwt" "$DO_QWT"  
+    printf "%-20s %s\n" "--qwt" "Build with Qwt"
```

Comment:
This looks wrong to me. It doesn't fit the pattern of other packages. Also, the next line looks like it should take 3 arguments and only 2 are given.

Reply:
qwt is required, so the [] part isn't needed (it should always be on). See bv_qt.sh.

Reply:
Ok. Makes sense. What about line 75? I see 3 %s in the first string and only 2 more strings after that. Should the last %s in the brackets be removed?

Reply:
Not sure, I was following the logic from qt and other required libs.
Perhaps the third argument should be "$USE_SYSTEM_QWT" ?
Other libraries that allow alt/system locations would need their usage text updated too.

Reply:
That seems correct.

Reply:
Sorry, could you clarify?

Reply:
printf "%-20s %s [%s]\n" "--alt-qwt-dir" "Use Qwt from an alternative directory" "$USE_SYSTEM_QWT"

That would be for line 75, not 74. Also, I don't think you need to change it throughout build_visit if that's a lot of work. I assume you just wanted to make some quick improvements and not take on a big project. I find that this review process sometimes ends up making a big deal out of something minor.

Reply:
Actually, that's not right either, since the third argument presumably isn't optional and it would be the path for the directory. Maybe the simplest thing to do is remove the " [%s]".